### PR TITLE
Shuttlemania

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

MORE SHUTTLES FOR THE SHUTTLEGOD.

1. Moves the docking ports around at CC, putting the escape pods above the emergency shuttle, rather than the other way around.

2. Ports the shuttle from this PR: https://github.com/tgstation/tgstation/pull/33227
It costs 2.500 credits.

3. Ports the bottom layer of the map in this PR: https://github.com/Yonaguni/EuropaStation/pull/674
Price is 50.000 credits.

4. Updates the purchase menu slightly by moving some newlines around

5. Adds "the ashes", a mismash of shuttles that happened by accident when we merged the previous shuttle PR. Price is 10.000 credits and an emag

Pics:

Makara:
image

Train:
image

Ashes:
image

#### Changelog

:cl:  Armhulen & Zuhayr
rscadd: More shuttles!
/:cl:

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -210,6 +210,18 @@
 	emag_buy = TRUE
 	credit_cost = 20000
 
+/datum/map_template/shuttle/emergency/makara
+	suffix = "makara"
+	name = "Makara Emergency Shuttle"
+	description = "A refitted submarine, now used for emergency travel"
```
Comment (Jc1469) [inline code/datums/shuttles.dm]: .

**Comment by ghost** [comment]
hey thanks for using my train but make sure you're mirroring my changes since I had a lot of trouble getting it working

**Comment by AsV9** [comment]
Tested it. Looks fine to me.

**Comment by Time-Green** [comment]
Could you perhaps surround the train tiles with thin windows? That way not literally fucking everyone gets sucked out if they try to move

**Comment by Amelia0010** [comment]
> Could you perhaps surround the train tiles with thin windows? That way not literally fucking everyone gets sucked out if they try to move

There are tiny fans under every single door. Every single one.

**Comment by Amelia0010** [comment]
> hey thanks for using my train but make sure you're mirroring my changes since I had a lot of trouble getting it working

Not entirely sure why yours didn't work, but this worked fine

**Comment by ghost** [comment]
oh i see why. your server doesn't use map merge

neat!

**Comment by Amelia0010** [comment]
We do if we change the maps, I just haven't been told to use it when I create them

**Comment by ShadowDeath6** [comment]
these look really big, do they fit on the escape dock without touching any part of the station?

**Comment by Amelia0010** [comment]
They fit on the escape dock, without touching anything except for the plating that normally surrounds the escape shuttle. Their size goes upwards, not downwards, and there's literally nothing but lattices and empty plating above it

**Comment by MistakeNot4892** [comment]
A lot of the turfs on the Makara map are there because there's no asteroid/debris shielding, you could probably trim it down to be less thicc without losing anything.

**Comment by Amelia0010** [comment]
But what if I like thicc walls? 🤤

**Comment by MistakeNot4892** [comment]
You do you, dude.

**Comment by AsV9** [comment]
Honestly I like the thicc walls on the makara shuttle.
This merge ready, nich?

**Comment by Amelia0010** [comment]
yep

**Pull request merged**